src/main/java/com/google/api/codegen/discogapic/transformer/java/JavaDiscoGapicRequestToViewTransformer.java

```diff
@@ -233,8 +233,8 @@ private StaticLangApiMessageView generateRequestClass(
       paramView.isRequired(false);
       paramView.canRepeat(false);
       paramView.fieldGetFunction(context.getDiscoGapicNamer().getResourceGetterName(param));
-      paramView.fieldSetFunction(context.getDiscoGapicNamer().getResourceSetterName(param));
-      paramView.properties(new LinkedList<StaticLangApiMessageView>());
+      paramView.fieldSetFunction(context.getDiscoGapicNamer().getResourceSetterName(param, false));
```

Comment:
Boolean parameters make calling code hard to read - could you either change the parameter type (e.g. make an enum), or make a separate function for repeated resource setter names, or something else?

Reply:
Done.